### PR TITLE
[Documentation:Developer] Preliminary steps for pushing to a fork

### PR DESCRIPTION
**Pull request by micpap25**
image

Updates the site as requested to reduce the chances of the commit history issue happening again

### CODE REVIEW

File: _docs/developer/getting_started/commit_to_PR_from_fork.md [hunk 1]
```diff
@@ -20,8 +20,14 @@ merging to the main branch.
 
 The instructions below are for command line use of git.
 
+1. **Before** you begin, confirm that your lcoal main branch of Submitty is
```
Comment (zacharymnp) [inline _docs/developer/getting_started/commit_to_PR_from_fork.md:23]: lcoal -> local

File: _docs/developer/getting_started/commit_to_PR_from_fork.md [hunk 2]
```diff
@@ -20,8 +20,14 @@ merging to the main branch.
...
+    up-to-date. Additionally, if the PR's branch is not up-to-date with Submitty's main,
+    there will be a button on its Github webpage that you can press to update it.
```
Comment (zacharymnp) [inline _docs/developer/getting_started/commit_to_PR_from_fork.md:25]: Github webpage -> page on Github (kind of personal preference, but I think it sounds better

File: _docs/developer/getting_started/commit_to_PR_from_fork.md [hunk 3]
```diff
@@ -20,8 +20,14 @@ merging to the main branch.
...
+    You may need to resolve some merge conflicts in the process of updating it.
```
Comment (zacharymnp) [inline _docs/developer/getting_started/commit_to_PR_from_fork.md:26]: remove 'some' (also kind of personal preference)

**Review by zacharymnp — CHANGES_REQUESTED** [review]

**Review by zacharymnp — APPROVED** [review]

File: _docs/developer/getting_started/commit_to_PR_from_fork.md (hunk 2 above)
Comment (micpap25) [reply _docs/developer/getting_started/commit_to_PR_from_fork.md]: does "button on its page on Github" sound odd?

**Review by bmcutler — APPROVED** [review]

**Pull request merged**